src/rgw/rgw_iam_policy.h

```diff
@@ -359,6 +360,18 @@ struct Condition {
     }
   };
 
+  struct ci_string_like : public std::binary_function<const std::string,
+                                                     const std::string,
+                                                     bool> {
+    bool operator ()(const std::string& s1,
+                    const std::string& s2) const {
+      if(fnmatch(s2.c_str(), s1.c_str(), 0) == 0){
```

Comment:
hi @cbodley  i change the function ci_string_like to string_like ,  i think  string_like should be case-sensitively